app/lib/connector-templates/updateConnectorTemplates.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+/*
+ * @typedef { {
+ *   id: string,
+ *   version?: number;
+ * } } Template
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const MARKET_PLACE_API_URL = 'https://marketplace.cloud.camunda.io/api/v1';
+
+const log = require('../log')('app:connector-templates');
+
+const connectorTemplatesFileName = '.camunda-connector-templates.json';
```

Comment:
The connectors will be stored in the existing user template path, per computer, in a single file, right?

This will make it hard to understand which connectors are fetched (as the file will be huge), but I guess this is also not our intention?

An alternative could be to store the connectors in a flat structure, similar to how they [are published](https://github.com/camunda/connectors/releases/download/8.5.4/connectors-bundle-templates-8.5.4.tar.gz), under a dedicated folder.

Reply:
Yes, all templates will be stored in a single file. Do you mean storing them as separate files? So for example

```
resources
├─ element-templates
  ├─ .camunda
    ├─ .asana.json
    ├─ ...
```

Reply:
Yes. How would that be relevant:

* If there was ever a reason for humans to manage these templates.

When would it not matter:

* If a human is not intended to _ever_ look into these files
* If I use custom connectors in my project(s) I can disable the flag and we would stop to load the built-in templates (otherwise folks would run into conflicts that they cannot fix).

Reply:
Yeah, so the idea is that this file is not touched by the user. With Cawemo we would put the fetched element templates in the `config` file which I decided against this time. Plus, you can disable them and then you won't see them.